satpy/readers/ahi_hsd.py

```diff
@@ -444,7 +465,8 @@ def read_band(self, key, info):
         res = xr.DataArray(res, attrs=new_info, dims=['y', 'x'])
 
         # Mask space pixels
-        res = self._mask_space(res)
+        if (self.mask_space):
```

Comment:
There is no need for extra parentheses here

Reply:
I've removed them now.